shared/js/media/jpeg_metadata_parser.js

```diff
@@ -305,7 +312,7 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
 
     var total = count * typesize[type];
     var valueOffset = total <= 4 ? offset + 8 :
-      data.getUint32(offset + 8, byteorder);
+        data.getUint32(offset + 8, byteorder) + startOffset;
```

Comment:
If I understand correctly, startOffset will always be 10, right?  The code is already full of "+ 10" constants. So I'd rather fix this by just adding a literal 10 here without using a variable.  That seems much simpler than declaring a variable, then later setting its value then still later using the value.

If I'm wrong, and it is possible that sometimes startOffset will be 0 or will be 20 or something, then this code looks good to me.